drivers/frequency/ltc6953/ltc6953.c

```diff
@@ -479,10 +479,100 @@ int ltc6953_enable_temp_stat(struct ltc6953_dev *dev, bool is_en)
 			      no_os_field_prep(LTC6953_TEMPO_MSK, (uint8_t)is_en));
 }
 
+/**
+ * @brief Read LTC6953 Check VCO Input stats.
+ * @param dev - The device structure.
+ * @param is_ok - Container variable for VCO input status.
+ * @return Returns 0 in case of success or negative error code.
+ */
+int ltc6953_vco_status(struct ltc6953_dev *dev, bool *is_ok)
+{
+	int ret;
+	uint8_t readval;
+
+	ret = ltc6953_read(dev, LTC6953_REG_VCO_STATUS, &readval);
+
+	if (ret)
+		return ret;
+
+	*is_ok = (bool)no_os_field_get(LTC6953_VCOOK_MSK, readval);
+
+	return 0;
+}
+
+/**
+ * @brief Read LTC6953 INVSTAT bit.
+ * @param dev - The device structure.
+ * @param status - Container variable for INVSTAT pin.
+ * @return Returns 0 in case of success or negative error code.
+ */
+int ltc6953_get_invstat(struct ltc6953_dev *dev, uint8_t *status)
```

Comment:
INVSTAT is single bit field, so this can be changed to bool instead. Doing so will remove the need to check for status.